### DISCUSSION TITLE
How to accept incoming imap connections on both mail.example.com and example.com and have valid certs for each ?

### DISCUSSION THREAD

**Discussion by shodanx2** [docker-mailserver/docker-mailserver#2864] [Q&A]
Hi,

I setup my email server using docker-compose.yml

With the entries

    hostname: mail
    domainname: example.com

Both mail.example.com and example.com A records point to the same address
In my /etc/letsencrypt I have recent certificates for both

However if I connect to example.com when using imap, I get a certificate invalid error.

I would like both of them to work, is that possible ?

**Comment by casperklein** [comment]
You can use a SAN or a wildcard certificate for that. Both are possible with let's encrypt.
